src/cnxml/se.tsx

```diff
@@ -496,6 +497,19 @@ function mediaItem(
     </Tag>
 }
 
+function processingInstruction(
+    node: ProcessingInstruction,
+    attrs: CommonAttrs,
+    children: JSX.Node,
+): JSX.Node {
+    return <pi
+        target={node.target}
+        value={node.value}
+        {...attrs}>
+        {children}
+    </pi>
```

Comment:
Following my above comment about not confusing processing instructions with elements, this should be

```
return { target: node.target, value: node.value }
```

Regardless, adding `{...attrs}` and `{children}` here is confusing, as they will be ignored.